cmd/minikube/cmd/start_flags.go

```diff
@@ -169,6 +170,7 @@ func initMinikubeFlags() {
 	startCmd.Flags().StringVarP(&outputFormat, "output", "o", "text", "Format to print stdout in. Options include: [text,json]")
 	startCmd.Flags().StringP(trace, "", "", "Send trace events. Options include: [gcp]")
 	startCmd.Flags().Int(extraDisks, 0, "Number of extra disks created and attached to the minikube VM (currently only implemented for hyperkit and kvm2 drivers)")
+	startCmd.Flags().Duration(certExpiration, time.Hour*24*365, "Duration until minikube certificate expiration, defaults to one year.")
```

Comment:
Should probably indicate to the user that hours is the largest supported unit so they don't try using a larger unit. And would probably be useful for people unfamiliar with duration format to give the an example format after the default.
```
defaults to one year (8760h).
```

Reply:
yep, done